src/helpers/env.ts

```diff
@@ -15,8 +16,10 @@ export default cleanEnv(process.env, {
   ECDSA_PRIVATE_KEY: str(),
   EDDSA_PRIVATE_KEY: str(),
   ETH_NETWORK: str({ default: ETH_NETWORK }),
+  ETH_POLYGON_NETWORK: str({ default: ETH_MUMBAI_NETWORK }),
   ETH_RPC: str({ default: ETH_RPC }),
   ETH_RPC_MAINNET: str({ default: ETH_RPC_MAINNET }),
+  ETH_RPC_POLYGON: str(),
```

Comment:
Should we set here the default value as in `ETH_RPC_MAINNET`?

Reply:
We don't have default RPC for Mumbai